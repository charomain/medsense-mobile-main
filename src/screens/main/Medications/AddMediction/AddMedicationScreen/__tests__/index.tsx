jest.mock('../../../../../../shared/components/Screen', () => {
  const RN = jest.requireActual('react-native');
  return {MedsenseScreen: RN.View};
});

import React from 'react';
import renderer from 'react-test-renderer';
import {
  createMockNavigationProps,
  MockAppProvider,
  mockSuccesfulResponse,
  wait,
} from '../../../../../../__mocks__/helpers';
import {AddMedicationScreen} from '../index';

test('renders correctly', async () => {
  mockSuccesfulResponse(200, []);

  const tree = renderer
    .create(
      <MockAppProvider>
        <AddMedicationScreen {...createMockNavigationProps()} />
      </MockAppProvider>,
    )
    .toJSON();

  await wait(1);

  expect(tree).toMatchSnapshot();
});
