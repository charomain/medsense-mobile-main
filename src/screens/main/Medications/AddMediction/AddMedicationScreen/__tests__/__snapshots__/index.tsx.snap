// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    Array [
      Object {
        "flex": 1,
      },
      Object {
        "backgroundColor": "#fff",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        false,
      ]
    }
  />
</RCTSafeAreaView>
`;
